bridge/client/app/_components/ktb-evaluation-details/ktb-evalution-details-utils.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  EvaluationResultType,
+  EvaluationResultTypeExtension,
+  IDataPoint,
+  IHeatmapTooltipType,
+} from '../../_interfaces/heatmap';
+import { ResultTypes } from '../../../../shared/models/result-types';
+import { Trace } from '../../_models/trace';
+import { IndicatorResult } from '../../../../shared/interfaces/indicator-result';
+
+export type SliInfo = {
+  score: number;
+  warningCount: number;
+  failedCount: number;
+  passCount: number;
+};
+
+const _resultType: { [key: string]: EvaluationResultType } = {
+  pass: ResultTypes.PASSED,
+  warning: ResultTypes.WARNING,
+  fail: ResultTypes.FAILED,
+  failed: ResultTypes.FAILED,
+  info: EvaluationResultTypeExtension.INFO,
+};
+
+export function getSliResultInfo(indicatorResults: IndicatorResult[]): {
+  score: number;
+  warningCount: number;
+  failedCount: number;
+  passCount: number;
+} {
+  return indicatorResults.reduce(
+    (acc, result) => {
+      const warning = result.status === ResultTypes.WARNING ? 1 : 0;
+      const failed = result.status === ResultTypes.FAILED ? 1 : 0;
+      return {
+        score: acc.score + result.score,
+        warningCount: acc.warningCount + warning,
+        failedCount: acc.failedCount + failed,
+        passCount: acc.passCount + 1 - warning - failed,
+      };
+    },
+    { score: 0, warningCount: 0, failedCount: 0, passCount: 0 } as SliInfo
+  );
+}
+
+const getSlisAndScore = (points: IDataPoint[], evaluation: Trace): IDataPoint[] => {
+  const totalScore: number = evaluation.data.evaluation?.indicatorResults
+    ? evaluation.data.evaluation?.indicatorResults.reduce((total: number, ir: IndicatorResult) => total + ir.score, 0)
+    : 1;
+  const scoreValue = evaluation.data.evaluation?.score ?? 0;
+
+  const indicatorResultToDataPoint = (indicatorResult: IndicatorResult): IDataPoint => {
+    const color = indicatorResult.value.success
+      ? _resultType[indicatorResult.status]
+      : EvaluationResultTypeExtension.INFO;
+    return {
+      xElement: evaluation.getHeatmapLabel(),
+      yElement: indicatorResult.value.metric,
```

Comment:
We also have the display name here. `indicatorResult.displayName || indicatorResult.value.metric`